### ISSUE TITLE
Seeming Incompatibility with Recursive Resources

### ISSUE THREAD

**Issue by D-Maxwell** [enjarai/shared-resources#46]
[logs](https://mclo.gs/4p4NDez)

Relevant error :
```
java.lang.NullPointerException: Cannot invoke "java.nio.file.Path.toFile()" because the return value of "java.util.function.Supplier.get()" is null
	at nl.enjarai.recursiveresources.compat.shared_resources.ExternalNestedFolderPackProvider.<init>(ExternalNestedFolderPackProvider.java:14)
	at net.minecraft.class_3283.handler$fdj01b$recursiveresources$onInit(class_3283.java:1545)
```

I may only bear suspicions, remaining devoid of any tangible pistes to follow.

**Comment by enjarai** [comment]
Most likely fixed in 1.8.4. Please let me know if that works.

**Comment by D-Maxwell** [comment]
I currently stand unable to test this for myself.

Would no other come to take care of this, this issue would not see closure prior to next Monday, in a week.

**Comment by D-Maxwell** [comment]
> Most likely fixed in 1.8.4. Please let me know if that works.

I am grateful for you have fixed this issue.
